### PR TITLE
adding kube_enforcer advance 6.0 - slk-30465 

### PR DESCRIPTION
**Pull request by KoppulaRajender**
adding kube_enforcer advance - slk-30465

### CODE REVIEW

File: orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      containers:
+        - name: kube-enforcer
+          image: registry.aquasec.com/kube-enforcer:6.0.RC1
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8449
+            - containerPort: 8442
+          env:
+            - name: AQUA_TOKEN
+              value: $token
```
Comment (guyyakir) [inline orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml:39]: should change to  valueFrom secret no ?

File: orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            - name: TLS_SERVER_KEY_FILEPATH
+              value: /certs/server.key
+            - name: AQUA_GATEWAY_SECURE_ADDRESS
+              value: aqua-gateway.aqua:8443
+            - name: AQUA_TLS_PORT
+              value: "8449"
+            - name: AQUA_KE_SERVER_PORT
+              value: "8442"
+            - name: CLUSTER_NAME
+              value: $clustername
```
Comment (guyyakir) [inline orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml:59]: should change $clustername no ?

File: orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml (hunk 1 above)
Comment (KoppulaRajender) [reply orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml]: yes, updated the PR

File: orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml (hunk 2 above)
Comment (KoppulaRajender) [reply orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml]: Thanks, updated PR with a default value.

**Pull request merged**